lib/airbrake-ruby/async_sender.rb

```diff
@@ -99,5 +101,16 @@ def spawn_worker
         end
       end
     end
+
+    def will_not_deliver(notice)
+      backtrace = notice[:errors][0][:backtrace].map do |line|
+        "#{line[:file]}:#{line[:line]} in `#{line[:function]}`"
+      end
+      @config.logger.error(
+        "#{LOG_LABEL} AsyncSender has reached its capacity of "                   \
+        "#{@unsent.max} and the following notice will not be delivered "          \
+        "Error: #{notice[:errors][0][:type]} - #{notice[:errors][0][:message]}\n" \
+        "Backtrace: \n" + backtrace.join("\n"))
+    end
```

Comment:
Could you please return `nil` explicitly from here, so it matches what we state on line 25?